### DISCUSSION TITLE
Wemos D1 Mini - JSON Configurator Question

### DISCUSSION THREAD

**Discussion by cgrylmz** [RavenSystem/esp-homekit-devices#1859] [Q&A]
Hello, i have a question; 
If i want to use Wemos D1 Mini, while using HAA Json Configurator, what should i select for sample config? 
Is it "Sonoff Standart Switch" ?

**Comment by paulzhdanov** [comment]
There is an example in wiki, just copypaste the code to configurator to see how it works (and - yes, if you wanna use it as a switch)

**Comment by cgrylmz** [comment]
I've configured d1 mini with this relay, 
![](https://electropeak.com/pub/media/catalog/product/cache/62c775a30744899c09966ae3c5e4ce4f/r/e/rel-01-015-1.jpg) 

i've used 8 pin connectors to connect them together.

is this json string is valid?

{{"c":{"l":2},"a":[{"t":1,"b":[{"g":1,"t":1}],"s":0,"i":3}]}

**Comment by cgrylmz** [comment]
didn't work, lol

**Comment by cgrylmz** [comment]
i thought i found the problem and used this code; 
{"c":{"l":2},"a":[{"0":{"r":[{"g":5,"v":0}]},"1":{"r":[{"g":5,"v":1}]},"t":1,"b":[{"g":15,"t":1}],"s":0,"i":3,"f0":[]}]}

But this time, i'm unable to pair the device with homekit, i couldn't find out what i did wrong about pairing, or what i did right at my first try haha 🗡️
